#Center of the rotated rectangle
geometry_msgs/Point centerPoint

#The width of the rotated rectangle
float64 width
#The height of the rotated rectangle
float64 height

#The angle of the rotated rectangle
float64 angle